lib/rawQueries.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const BuildFactoryQueries = {
+    statusesQuery:
+        `SELECT r."id", r."jobId", j."name" as "jobName", r."status", r."startTime", r."endTime"
+        FROM (SELECT "id", "jobId", "status", "startTime", "endTime",
+        RANK() OVER ( PARTITION BY "jobId" ORDER BY "id" DESC ) AS rank
+        FROM builds WHERE "jobId" in (:jobIds)) r
+        INNER JOIN jobs as j ON j."id" = r."jobId"
+        WHERE rank > :offset AND rank <= :maxRank
+        ORDER BY r."jobId", r."id" DESC`,
+
+    statusesQueryMySql:
+        `SELECT r.id, r.jobId, j.name as \`jobName\`, r.status, r.startTime, r.endTime
+        FROM (SELECT id, jobId, status, startTime, endTime,
+        RANK() OVER ( PARTITION BY jobId ORDER BY id DESC ) AS \`rank\`
+        FROM builds WHERE jobId in (:jobIds)) r
+        INNER JOIN jobs as j ON j.id = r.jobId
+        WHERE \`rank\` > :offset AND \`rank\` <= :maxRank
```

Comment:
@InderH for mysql we need ` prefix ?

Reply:
ignore, got it

Reply:
I just used a formatter to read, its essentially 
```sql
SELECT "id", 
       "jobid", 
       "status", 
       "starttime", 
       "endtime" 
FROM   (SELECT "id", 
               "jobid", 
               "status", 
               "starttime", 
               "endtime", 
               Rank() 
                 OVER ( 
                   partition BY "jobid" 
                   ORDER BY "id" DESC ) AS rank 
        FROM   builds 
        WHERE  "jobid" IN ( :jobIds )) AS R 
WHERE  rank > :offset 
       AND rank <= :maxRank 
ORDER  BY "jobid", 
          "id" DESC 
```